buildspec.yml

```diff
@@ -0,0 +1,42 @@
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      nodejs: 14
```

Comment:
Can we bump this up to 18? Support for node 14 ends this April.

Reply:
Only Node versions supported by the pipeline are 12 and 14

Reply:
We could revisit this soon when we have to upgrade from Solutions pipeline v5 to v6